mempool/reactor.go

```diff
@@ -17,7 +18,7 @@ import (
 const (
 	MempoolChannel = byte(0x30)
 
-	protoOverheadForTxMessage = 4
+	protoOverheadForTxsMessage = 4
```

Comment:
@marbar3778 how is this computed? can you tell? thanks

Reply:
There is a 2 byte overhead when encoding a ` bytes tx = 1;` in an `oneof`. In the past with amino it was larger due to interface encoding overhead. I noticed in places previously we added an extra byte or two to be safe when calculating overhead, so I left it as 4 here. We can drop it lower.